### PR TITLE
Readme proposal

### PR DESCRIPTION
**Pull request by Legersky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,85 @@
...
+
+## Documentation
+
+The documentation of the `main` branch is available [online](https://pyrigi.github.io/PyRigi/).
+For compiling it locally,
+see the [development guide](https://pyrigi.github.io/PyRigi/development/howto).
+
+An important part of the documentation is the mathematical background.
+We specify the outputs of the methods in the package
+by giving rigorous mathematical definitions.
```
Comment (matthiashimmelmann) [inline README.md:37]: "by providing rigorous"

File: README.md [hunk 2]
```diff
@@ -2,9 +2,85 @@
...
+
+## Questions and feature requests
+
+We have a [Zulip chat](https://pyrigi.zulipchat.com),
+where you can ask questions or propose new functionality.
+If you want to get access to it, please, send an email to
+[this address](mailto:external.dc4f45edef70cb7e0c621ad50377d9f1.show-sender.include-footer@streams.zulipchat.com).
+You can also use the [GitHub Discussions](https://github.com/PyRigi/PyRigi/discussions).
+
+To report bugs, please, create an [issue](https://github.com/PyRigi/PyRigi/issues/new/choose).
```
Comment (matthiashimmelmann) [inline README.md:47]: "To report bugs or ask for new features, please"

File: README.md [hunk 3]
```diff
@@ -2,9 +2,85 @@
...
+
+## Contributing
+
+We appreciate contributions!
+Do you have a new research result
```
Comment (matthiashimmelmann) [inline README.md:52]: Doesn't have to be new right?

File: README.md [hunk 4]
```diff
@@ -2,9 +2,85 @@
...
+about rigidity or flexibility of bar-joint frameworks
+that could be implemented in the package?
```
Comment (matthiashimmelmann) [inline README.md:54]: "that could be implemented[?]"

File: README.md [hunk 5]
```diff
@@ -2,9 +2,85 @@
...
+[Let us know](https://github.com/PyRigi/PyRigi/issues/new/choose)!
+Or even better, implement it and contribute to the package!
+
+Except for coding, you can also help for instance
```
Comment (matthiashimmelmann) [inline README.md:58]: Besides coding,

File: README.md [hunk 6]
```diff
@@ -2,9 +2,85 @@
...
+by extending the mathematical documentation or
+creating tutorials.
+
+If you want to contribute, please,
+read the [development guide](https://pyrigi.github.io/PyRigi/development/howto)
+and [contact us](mailto:external.dc4f45edef70cb7e0c621ad50377d9f1.show-sender.include-footer@streams.zulipchat.com).
+## License
+
+The package is licensed under [MIT license](https://github.com/PyRigi/PyRigi/blob/main/LICENSE).
```
Comment (matthiashimmelmann) [inline README.md:67]: "[the] MIT License"

**Review by matthiashimmelmann — CHANGES_REQUESTED** [review]
I really like how you structured the Readme. There are just a few minor semantic suggestions for it, but besides them I think you did a great job and it already looks really professional!

**Comment by Legersky** [comment]
Thanks, I like your suggestions! Feel free to incorporate them if you want, or I can do it later. See Zulip for the Zulip question.

**Review by matthiashimmelmann — APPROVED** [review]
Approved.

**Pull request merged**